dizoo/mujoco/config/hopper_trex_onppo_config.py

```diff
@@ -57,16 +63,21 @@
         eval=dict(evaluator=dict(eval_freq=5000, )),
     ),
 )
-hopper_trex_ppo_default_config = EasyDict(hopper_trex_ppo_default_config)
-main_config = hopper_trex_ppo_default_config
+hopper_trex_onppo_config = EasyDict(hopper_trex_onppo_config)
+main_config = hopper_trex_onppo_config
 
-hopper_trex_ppo_create_default_config = dict(
+hopper_trex_onppo_create_config = dict(
     env=dict(
         type='mujoco',
         import_names=['dizoo.mujoco.envs.mujoco_env'],
     ),
     env_manager=dict(type='subprocess'),
     policy=dict(type='ppo', ),
 )
-hopper_trex_ppo_create_default_config = EasyDict(hopper_trex_ppo_create_default_config)
-create_config = hopper_trex_ppo_create_default_config
+hopper_trex_onppo_create_config = EasyDict(hopper_trex_onppo_create_config)
+create_config = hopper_trex_onppo_create_config
+
+
+if __name__ == "__main__":
```

Comment:
add line # or you can enter `ding -m serial_trex_onpolicy -c hopper_trex_onppo_config.py -s 0`